custom_metrics/fugu-apis.js

```diff
@@ -0,0 +1,213 @@
+const responseBodies = $WPT_BODIES;
+
+// Data created via a little helper spreadsheet:
+// https://docs.google.com/spreadsheets/d/1ndxh6sl0fSOLHFlMdSiLPqmGPMDVJqkJFYnQ0Hsmhwo/edit?usp=sharing
+const patterns = {
```

Comment:
Love this approach! I think this is a nice and extensible system for organizing all of the different usage patterns.

Reply:
I will try to make the sheet create the whole source code.